eta-meeting-organizer-backend/src/main/java/hu/flowacademy/meetingorganizer/service/ReservationService.java

```diff
@@ -73,6 +73,9 @@ public Reservation updateReservation(Long id, ReservationDTO reservationInput) {
   }
 
   private void validateReservation(ReservationDTO input) {
+    if(!reservationRepository.findAllByMeetingRoomIdInInterval(input.getMeetingRoomId(), input.getStartingTime(), input.getEndingTime()).isEmpty()) {
```

Comment:
Az userId checkolásnál még meg is értem, hogy nem szervezzük ki változóba, de ez így szerintem nem túl olvasható. 
Egyébként ha a lekérdezést csak arra használjuk, hogy eldöntsük, hogy van-e eredményhalmaza, miért nem olyan lekérdezést írunk meg, amiből egy érték jön vissza (azaz nem kell az egész eredményhalmazt memóriában tárolni) és az alapján hozunk egy bináris döntést? 
Edit: hogy egy kicsit konkrétabb legyek: írjunk olyan lekérdezést inkább, ami egy countot ad vissza, és ha az 0, akkor dobjunk exceptiont.

Reply:
Azt kérte Robi, hogy ne legyenek külön változók a paraméterek és, hogy egyből menjen a függvényhívás az if-be, ezt ki is javítottam a kérésére. Előtte külön változóba voltak a paraméterek. Köszönöm az észrevételt, átírtam a query-t count-ra, ahogy kérted, most ha az értéke nagyobb, mint nulla akkor fog exceptiont dobni.